### ISSUE TITLE
Option for SPECTER2 embeddings?

### ISSUE THREAD

**Issue by japhba** [augustwester/searchthearxiv#3]
Hi,

I recently started to use Semantic Scholar's SPECTER2 model to create visualisations of BibTex files. The model is specialised to scientific work, so it seems a good option for the proximity search that searchthearxiv.com offers. While there are gaps in what papers have an associated embedding in their database, its scope extends beyond the ArXiv. 

I was thinking to create a service "What was that paper again?" that would 
- Take in a description of the user with length of one to ten sentences
- Embed this description using SPECTER2
- Do a proximity search 
- Return matching candidates

In summary, the advantages would be
- broader scope beyond ArXiv
- potentially longer queries
- a more accurate backend model

It would be exciting to see this functionality integrated into SearchTheArXiv, I would be very willing to do a prototype!

**Comment by augustwester** [comment]
Thanks for the suggestion! However, I think this is outside the scope of searchthearxiv.com, which is intentionally low-maintenance, ultra low-cost (i.e. basically free to run), and limited to ML papers. How would searchthearxiv.com differ from Semantic Scholar if we implemented what you describe?
